routes.js

```diff
@@ -47,4 +47,12 @@ routes.post(
   shoppingListProductsController.create
 );
 
+routes.get("/health", (req, res) => {
+  const applicationName = config.APP_NAME;
+
+  res.json({
+    message: `Aplicação: ${applicationName} - Tá rodando no servidor`,
```

Comment:
Eu usaria uma mensagem menor se isso for rodar num load balance terá um volume grande de dados trafegados de baixa relevância. Um simples retorno vazio com 204 já seria suficiente para checar a saúde de um serviço.
Um endpoint de status poderia discriminar melhor como está cada recurso que demande ter sua saúde aferida.

Reply:
Fiz isso!